packages/remix-dev/cli/run.ts

```diff
@@ -186,7 +186,9 @@ export async function run(argv: string[] = process.argv.slice(2)) {
       // as possible, but inquirer will allow users to retry input rather than
       // stop the process.
       if (flags.template) {
-        await validateTemplate(flags.template);
+        await validateTemplate(flags.template, {
+          githubToken: process.env.GITHUB_TOKEN,
+        });
```

Comment:
I'm happy to revert this and read the process value from inside `validateTemplate`, but I thought I'd be consistent with `createApp` if possible.

Reply:
@mcansh will give you feedback on this item